src/app/providers/splitsProvider.tsx

```diff
@@ -0,0 +1,41 @@
+"use client";
+
+import { SplitsClientConfig, SplitsProvider } from "@0xsplits/splits-sdk-react";
+import { useEffect, useState } from "react";
+import { useChainId, usePublicClient, useWalletClient } from "wagmi";
+
+type Props = {
+  children?: React.ReactNode;
+};
+
+export const PaymentSplitsProvider = ({ children }: Props) => {
+  const [splitsConfig, setSplitsConfig] = useState<SplitsClientConfig | null>(
+    null,
+  );
+  const { data: walletClient, status, error } = useWalletClient();
+  const publicClient = usePublicClient();
+  const chainId = useChainId();
+
+  useEffect(() => {
+    if (status == "success") {
```

Comment:
if (status == "success" && chainId && chainId typeof number)

Reply:
chainId is a number no need for typeof number check